graylog2-web-interface/src/views/components/widgets/WidgetFrame.tsx

```diff
@@ -99,6 +99,19 @@ const WidgetWrap = styled.div(
         padding: 0 5px;
       }
     }
+
+    .widget-actions-menu,
+    .widget-drag-handle {
+      opacity: 0;
+      pointer-events: none;
+      transition: opacity 0.2s;
+    }
+
+    &:hover .widget-actions-menu,
+    &:hover .widget-drag-handle {
```

Comment:
Let's move these class names into a variable, to avoid regressions when renaming it in just one place?